roi-hunter-easy-for-woocommerce.php

```diff
@@ -15,7 +15,7 @@
 define( 'RH_EASY_DIR', plugin_dir_path( __FILE__ ) );
 define( 'RH_EASY_URL', plugin_dir_url( __FILE__ ) );
 define( 'RH_EASY_BASENAME', plugin_basename( __FILE__ ) );
-define( 'RH_EASY_VERSION', '1.0.3' );
+define( 'RH_EASY_VERSION', '1.0.5' );
```

Comment:
neměla by to být 1.0.4? nebo v changelogu přidat poznámku k 1.0.4?

Reply:
Čekal jsem, že se nejdřív mergne předchozí pull request. Rebasnuto, vyřešeno...